ForageBuddy/Form1.cs

```diff
@@ -11,33 +14,81 @@ public partial class Form1 : Form
         private string _folderPath;
         private List<string> _filesInScreenshotDirectory;
         private Dictionary<string, PlayerScore> _playerScores;
-
+        
+        private readonly ILogger _logger;
         private readonly ImageParser _imageParser;
+        private readonly string _logFile = "log.txt";
 
         public Form1()
         {
             _imageParser = new ImageParser();
             InitializeComponent();
-            lbScores.Items.Insert(0, "Please Specify Screenshot folder");
+            lbScores.Items.Insert(0, "Please specify a screenshot folder.");
+
+            var logFolder = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "logs");
+
+            // Check for logFolder, create if missing
+            if (!Directory.Exists(logFolder))
+                Directory.CreateDirectory(logFolder);
+
+            // Check for logFile, create new name if one already exists
+            if (File.Exists(Path.Combine(logFolder, _logFile)))
+            {
+                var i = 1;
+                
+                _logFile = _logFile.Insert(_logFile.IndexOf("."), $" ({i})");
+                
+                while (File.Exists(Path.Combine(logFolder, _logFile)))
+                {
+                    i++;
+                    _logFile = _logFile.Replace($"{i - 1}", $"{i}");
+                }
+            }
+            
+            // Create logger to record program activity
+            _logger = new LoggerConfiguration()
+                .WriteTo.File(Path.Combine(logFolder, _logFile))
+                .CreateLogger();
+            
+            _logger.Information("Application started.");
+            _logger.Information($"Logging at: {Path.Combine(logFolder, _logFile)}");
         }
 
         private void btnFolderSelect_Click(object sender, EventArgs e)
         {
-            if (fbdScreenieFolder.ShowDialog() != DialogResult.OK) return;
+            _logger.Information($"\"{sender.ToString().Substring(sender.ToString().IndexOf("Text: ") + 6)}\" button pressed.");
+            
+            // Select an output folder
+            _logger.Information("Selecting output folder...");
+            
+            if (fbdScreenieFolder.ShowDialog() != DialogResult.OK)
+            {
+                _logger.Information("Folder not selected. Returning.");
```

Comment:
Don't need to say what is happening - just saying folder not selected would be sufficient.